templates/server/src/main/resources/archetype-resources/__batch__/src/main/java/__packageInPathFormat__/SpringBootBatchApp.java

```diff
@@ -13,10 +13,8 @@
 /**
  * {@link SpringBootApplication} for running this batch.
  */
-@SpringBootApplication(exclude = { EndpointAutoConfiguration.class, SecurityAutoConfiguration.class,
-SecurityFilterAutoConfiguration.class, })
+@SpringBootApplication
 @EntityScan(basePackages = { "${package}" }, basePackageClasses = { AdvancedRevisionEntity.class })
-@EnableGlobalMethodSecurity(jsr250Enabled = false)
```

Comment:
So we are disabling autorization in the batch?
Usually in my projects we did a login from the batch launcher.
If that is currently not present at least better to have a batch that has all permissions than none ;)